renpy/text/text.py

```diff
@@ -352,6 +352,14 @@ def bounds(self, glyphs, bounds, layout):
         fo = font.get_font(self.font, self.size, self.bold, self.italic, 0, self.antialias, self.vertical, self.hinting, layout.oversample)
         return fo.bounds(glyphs, bounds)
 
+    @property
```

Comment:
I don't think a property is the right way to do this. Let's have a set_font method, which would be used in TextSegment.take_style, and then again in the implementation of the font tag.